test/onigumo_cli_test.exs

```diff
@@ -33,15 +34,18 @@ defmodule OnigumoCLITest do
     end
 
     test("run CLI with no arguments") do
-      assert_raise(MatchError, fn -> Onigumo.CLI.main([]) end)
+      output = capture_io(fn -> Onigumo.CLI.main([]) end)
+      assert String.starts_with?(output, "Usage: ./onigumo ")
     end
 
     test("run CLI with more than one argument") do
-      assert_raise(MatchError, fn -> Onigumo.CLI.main(["Downloader", "Parser"]) end)
+      output = capture_io(fn -> Onigumo.CLI.main(["Downloader", "Parser"]) end)
+      assert String.starts_with?(output, "Usage: ./onigumo ")
```

Comment:
These two tests for an invalid number of arguments can be amended in a follow-up pull request, adding the _MatchError_.